online-store/src/app/app.ts

```diff
@@ -0,0 +1,278 @@
+import { addFiltersToUrl, sortingArray, minPriceFunc, maxPriceFunc, updatingShoppingCart, parseStringAndAddToObject  } from "../functions/functions";
+import { productsArray } from "../app/products";
+import { ProductDescription, ProductFilters } from "../types/types";
+import { createTable, } from "../app/rendering";
+import { slider } from "../app/slider";
+
+let countryArray: string [] = [];
+let brandArray: string [] = [];
+let sortArray = productsArray;
+const currentUrl = window.location.href;
+//let sortingValue: string = 'priceAscending';
+let filtersObj: ProductFilters = {
```

Comment:
Instead of using the let keyInstead of using the let keyword, use const where possible to make variables immutable.word, use const where possible to make variables immutable.